### PR TITLE
docs: invite @inottn to Rspack core team

### PR DESCRIPTION
**Pull request by chenjiahan**
### New team member

Thanks @inottn for your contribution to Rspack stack!

I am delighted to invite you to join the Rspack core team! Your outstanding contributions to the project have truly impressed us. We'd love to have you on board to help shape the future of Rspack stack. ❤️

We have prepared a Rspack gift box for you. If it's convenient, please contact me via email: <jait.chen@foxmail.com>

Contributions:

- https://github.com/web-infra-dev/rspack/commits?author=inottn
- https://github.com/web-infra-dev/rsbuild/commits?author=inottn
- https://github.com/web-infra-dev/rspack/issues?q=commenter:inottn
- https://github.com/web-infra-dev/rsbuild/issues?q=commenter:inottn

## Checklist

- [ ] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

File: website/components/RandomMemberList.tsx [hunk 1]
```diff
@@ -165,6 +165,11 @@ const coreTeam: Member[] = [
     x: 'https://x.com/dalaoshv',
     desc: 'Rspack stack contributor',
   },
+  {
+    id: 'inottn',
+    avatar: 'https://avatars.githubusercontent.com/u/18509404?v=4',
+    desc: 'Rspack stack contributor, Vant core team member',
```
Comment (chenjiahan) [inline website/components/RandomMemberList.tsx:171]: @inottn do you want to add your X account here?

Comment (inottn) [reply website/components/RandomMemberList.tsx:171]: Sure! https://x.com/inorr_r 😄

**Review by LingyuCoder — APPROVED** [review]

**Comment by inottn** [comment]
I just came across your email from a few months ago while cleaning out my inbox. I apologize for not seeing it sooner, as my inbox was flooded with messages at the time. 😄

**Comment by chenjiahan** [comment]
Haha, never mind 😄

**Review by hardfist — APPROVED** [review]

**Pull request merged**